packages/ui/src/Body/index.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import { body } from '../utils/typography';
+
+/**
+ * Body text used throughout most of our UIs.
+ *
+ * Renders a `<span />` by default; pass the `as` prop to use a different HTML
+ * element with the same styling.
+ */
+export const Body = styled.span`
+  ${body}
+`;
```

Comment:
thought: it was unexpected to see components like `Body`, `Technical` and others being simple texts. The DX of this approach is ultimately not the best. Imagine a common routine of changing one text type to another: users will have to import a new text type, change the usage, and remove the old import.

main suggestion of this PR: simply create a `Typography` component with `type: 'body' | 'h1' | 'h2' | 'large' | etc` property. It will still be lightweight – there's not much logic or styles applied

Reply:
Ugh so, that's actually what I was doing initially, for the same reason. Then I split them out (https://github.com/penumbra-zone/web/pull/1489/commits/7740d3f7ffc0fdb484d74fafe55120d77676ebb5) into separate components because I realized it would get super verbose to have to wrap a little bit of text in `<Typography type='technical'>{text}</Typography>` instead of just `<Technical>{text}</Technical>`.

I'm not 100% married to the approach that I have here, but I do feel like it's easier to use (and read) than the more verbose approach.

Although... maybe this could be mitigated by just using a more abbreviated component name? e.g., `Text` (like in React Native). And instead of a `type` prop, it would allow you to set a single boolean prop for the type of text you want to render, and it'd use TypeScript to enforce that you'd only pass one, e.g.:

```tsx
<Text h1>This is an H1</Text>
<Text body>This is body text</Text
<Text h2 technical>This will throw a TypeScript error</Text>
```

This would also fix the issue you mentioned of importing a new type and removing the old import.

I think I'm liking this solution! I'm going to implement it and request re-review.

Reply:
@VanishMax OK, the new `<Text />` component is ready for your review. Please give this PR the 👍🏻 if it looks good to you.